packer/build-rhel-7.json

```diff
@@ -144,6 +144,15 @@
         "amazon-ebs"
       ]
     },
+    {
+      "type": "shell",
```

Comment:
Replace all theses with a single packer/linux/install-dotnet.sh script similar to the install-cmake.sh script. This way there is only one place to modify this script in the future. You can then call out this script in the packer just below the `"linux/install-cmake.sh"` line.

Reply:
I was just thinking this myself, once I noticed there were shell scripts in there already.